test/support/in_memory.ex

```diff
@@ -100,6 +100,10 @@ defmodule Hunter.Api.InMemory do
     %Hunter.Status{}
   end
 
+  def reblogged_by(_, _) do
```

Comment:
Could you please add a fixture, then include this in the list at the beginning of this file?